build/uno.winui.common.targets

```diff
@@ -8,6 +8,18 @@
 		<_IsUnoPlatform>true</_IsUnoPlatform>
 	</PropertyGroup>
 
+	<!-- Feature configuration -->
+	<PropertyGroup>
+		<UnoDragDropExternalSupport Condition="'$(UnoDragDropExternalSupport)'==''">true</UnoDragDropExternalSupport>
+	</PropertyGroup>
+	<ItemGroup>
+		
+		<RuntimeHostConfigurationOption Include="Windows.ApplicationModel.DataTransfer.DragDrop.ExternalSupport"
+										Condition="'$(UnoDragDropExternalSupport)' != ''"
```

Comment:
This condition is always true, I think? In line 13, we set it to true if it was empty.